graphql/resolvers.go

```diff
@@ -1882,13 +1885,6 @@ func (r *mutationResolver) AbortTask(ctx context.Context, taskID string) (*restM
 	if t == nil {
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
 	}
-	p, err := r.sc.FindProjectById(t.Project)
```

Comment:
I realize this change isn't necessary, so I can remove it if you object, but this project isn't used / isn't necessary for an abort so I don't see why we need to do this.